articles/architecture-scenarios/_includes/_authentication/_introduction.md

```diff
@@ -32,3 +34,5 @@ Allowing users to sign in via user ID/password credentials means that you're not
 If you have an existing legacy identity store, you’ll also want to see [User Migration](/architecture-scenarios/implementation/${platform}/${platform}-provisioning#user-migration). This section discusses the advantages of migrating to Auth0’s managed identity storage in terms of safety and security.
 
 For customer facing applications, OpenID Connect ([OIDC](/protocols/oidc)) is the most frequently used industry standard protocol, and OIDC has first-class citizen support in Auth0. Auth0 provides support for various different approaches for integrating various different applications, so you'll want to see the section on [application integration](#application-integration) for the information you'll need to make an informed choice. 
+
+When calling one API from another API, cron jobs, report generators, or continuous integration/delivery systems, you will need a way to authorize the application instead of a user.  This requires some [machine-to-machine (m2m) authentication](#m2m-authentication).
```

Comment:
```suggestion
When calling one API from another API, or from any situation where there is no authenticated user context - such as one or more cron jobs, report generators, or continuous integration/delivery systems - you will need a way to authorize the _application_ instead of a _user_. This is referred to as [Machine-to-Machine (M2M) authentication](#m2m-authentication).
```